### PR TITLE
Add Centos/Fedora/RHEL Install Notes

### PR DESCRIPTION
**Pull request by Rosiak**

### CODE REVIEW

**Comment by maxpoulin64** [comment]
Looks fine to me, :+1: .

Maybe you could condensate the yum commands into one as well, just like the Debian/Ubuntu variant? It's a detail, but it makes the instructions be smaller.

**Comment by Rosiak** [comment]
@maxpoulin64 
Done, can't consolidate it more than this hence we need epel before asking for nodejs/npm.

**Comment by maxpoulin64** [comment]
@Rosiak Sorry to be strict, can you also please squash those commits together to keep our commit history as clean as possible? (`git rebase -i HEAD^^` and set the second commit to `squash`, then force push the branch).

**Comment by MaxLeiter** [comment]
maybe also include a note that newer fedora users should use dnf instead of yum?

**Comment by Rosiak** [comment]
Let me know if this is ok.

File: _docs/getting_started/install.md [hunk 1]
```diff
@@ -34,6 +34,25 @@ Install The Lounge:
 $ sudo npm install -g thelounge
 ```
 
+## CentOS / Fedora / RHEL
+
+### Step 1:
+
+Install the requirements using yum(If you are using a newer Fedora version, please use dnf instead):
```
Comment (MaxLeiter) [inline _docs/getting_started/install.md]: Maybe add a Soave between yum and (? Also, “newer version of fedora, please use dnf”) maybe?

**Comment by Rosiak** [comment]
@MaxLeiter 
Changed.

**Comment by MaxLeiter** [comment]
Looks good to me :+1:

**Pull request merged**